Modules/quote/Model/Quote.php

```diff
@@ -44,35 +48,48 @@ public function getAllInfo($id_space, $id) {
         $sql = "SELECT * FROM qo_quotes WHERE id=? AND id_space=? AND deleted=0";
         $data = $this->runRequest($sql, array($id, $id_space))->fetch();
 
-        if ($data['id_user'] != 0) {
-            $modelUser = new CoreUser();
-            $modelUserClient = new ClClientUser();
-            $modelClient = new ClClient();
+        $modelUser = new CoreUser();
+        $modelUserClient = new ClClientUser();
+        $modelClient = new ClClient();
 
+        if ($data['id_user'] == 0) {
+            // quote edited with "new user quote"
+            $data['client'] = $modelClient->get($id_space, $data["id_client"]);
```

Comment:
what if id_client not present or empty? or 0 ?

Reply:
so it selects the first client of the list ;). Ok, I change that

Reply:
Actually, that's ok:
ClClient::get($id_space, $id) returns an empty client if `!$id` (`"id" => 0, "name" => ""`, etc...)